snapcraft/linters/library_linter.py

```diff
@@ -101,6 +104,20 @@ def run(self) -> List[LinterIssue]:
 
         return issues
 
+    def _find_deb_package(self, library_name):
+        for path in glob.glob("/usr/lib/**", recursive=True):
+            if os.path.basename(path) == library_name:
+                try:
+                    output = subprocess.run(
+                        ["dpkg", "-S", path], check=False, stdout=subprocess.PIPE
```

Comment:
why `check=False`

Reply:
I originally wanted to manage separately the case when the call failed and the case where the file doesn't belong to a .deb package. But you are right: it is simpler this way.